packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -9871,6 +9871,45 @@ void main() {
     expect(rectWithText.height, greaterThan(100));
   });
 
+  testWidgets('Placeholder is baseline aligned with text', (WidgetTester tester) async {
```

Comment:
Nit: Would it be useful to test the case where the EditableText font is larger than the placeholder font?